lib/user_interface.rb

```diff
@@ -54,6 +62,14 @@ def ask_for_fields
     end
   end
 
+  def display_contact(contact_hash)
+    max_field_name = FIELDS_TO_FIELD_NAMES.values.max(1) { |a, b| a.length <=> b.length }[0]
```

Comment:
I would call this `longest_display_name` rather than `max_field_name` 

Also you can simplify this if you use [`max_by`](https://ruby-doc.org/core-2.7.1/Enumerable.html#method-i-max_by) instead of `max`:

```ruby
longest_display_name = FIELDS_TO_DISPLAY_NAMES.values.max_by(&:length)
```

it allows you to select the criteria for `max`, in this case length of the word.